<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SolarMonitorClass</class>
 <widget class="QMainWindow" name="SolarMonitorClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>MS Sans Serif</family>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>SolarMonitorClient</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="company_logo">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>70</y>
      <width>200</width>
      <height>220</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(&quot;./img/logo_eng.png&quot;) no-repeat center center stretch; border-radius: 0px; background-color: rgba(0, 0, 0, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="Power_groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>386</y>
      <width>230</width>
      <height>260</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:white;background-color: rgba( 255, 255, 255, 225);border-radius: 25px</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QtCharts::QChartView" name="PowerChart_widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>230</width>
       <height>260</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(&quot;&quot;); background-color: rgba( 255, 255, 255, 0 )</string>
     </property>
     <widget class="QLabel" name="label_Power_title">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>10</y>
        <width>191</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Current Power</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Env_groupBox">
    <property name="geometry">
     <rect>
      <x>319</x>
      <y>335</y>
      <width>480</width>
      <height>310</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:white;background-color: rgba( 255, 255, 255, 225);border-radius: 25px</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_static_EnvTitle">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>15</y>
       <width>480</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(&quot;&quot;); border-radius: 0px; background-color: rgba(0, 0, 0, 0)</string>
     </property>
     <property name="text">
      <string>Environmental savings</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_CO_Total">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>230</y>
       <width>151</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>28</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(&quot;&quot;); border-radius: 0px; background-color: rgba(0, 0, 0, 0)</string>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_static_EnvTotal">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>230</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(&quot;&quot;); border-radius: 0px; background-color: rgba(0, 0, 0, 0)</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="text">
      <string>Total:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_static_EnvToday">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>170</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(&quot;&quot;); border-radius: 0px; background-color: rgba(0, 0, 0, 0)</string>
     </property>
     <property name="text">
      <string>TODAY:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_CO2_Today">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>160</y>
       <width>151</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(&quot;&quot;); border-radius: 0px; background-color: rgba(0, 0, 0, 0)</string>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_PlantedTrees">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>291</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(&quot;&quot;); border-radius: 0px; background-color: rgba(0, 0, 0, 0)</string>
     </property>
     <property name="text">
      <string>CO2 REDUCTION IS EQUAL TO %s PLANTED TREES</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="env_logo">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>100</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(&quot;./img/env_logo.png&quot;); border-radius: 0px; background-color: rgba(0, 0, 0, 0)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_CO_Total_unit">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>241</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(&quot;&quot;); border-radius: 0px; background-color: rgba(0, 0, 0, 0)</string>
     </property>
     <property name="text">
      <string>KG</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_CO2_Today_unit">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>170</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(&quot;&quot;); border-radius: 0px; background-color: rgba(0, 0, 0, 0)</string>
     </property>
     <property name="text">
      <string>KG</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Charts_groupBox">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>55</y>
      <width>400</width>
      <height>590</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:white;background-color: rgba( 255, 255, 255, 225);border-radius: 25px</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QtCharts::QChartView" name="MonthChart_widget" native="true">
     <property name="geometry">
      <rect>
       <x>-20</x>
       <y>260</y>
       <width>441</width>
       <height>341</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(&quot;&quot;); background-color: rgba( 255, 255, 255, 0 )</string>
     </property>
    </widget>
    <widget class="QtCharts::QChartView" name="HourChart_widget" native="true">
     <property name="geometry">
      <rect>
       <x>-20</x>
       <y>-20</y>
       <width>441</width>
       <height>321</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(&quot;&quot;); background-color: rgba( 255, 255, 255, 0 )</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Eng_groupBox">
    <property name="geometry">
     <rect>
      <x>319</x>
      <y>55</y>
      <width>480</width>
      <height>222</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:white;background-color: rgba( 255, 255, 255, 225);border-radius: 25px</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_static_EnergyTotal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>230</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(&quot;&quot;); border-radius: 0px; background-color: rgba(0, 0, 0, 0)</string>
     </property>
     <property name="text">
      <string>Total:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_TotalEnergy">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>160</y>
       <width>141</width>
       <height>50</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777208</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(&quot;&quot;); border-radius: 0px; background-color: rgba(0, 0, 0, 0)</string>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_static_EnergyTitle">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>480</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(&quot;&quot;); border-radius: 0px; background-color: rgba(0, 0, 0, 0)</string>
     </property>
     <property name="text">
      <string>Produced energy</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_TodayEnergy">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>80</y>
       <width>141</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(&quot;&quot;); border-radius: 0px; background-color: rgba(0, 0, 0, 0)</string>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_static_EnergyToday">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(&quot;&quot;); border-radius: 0px; background-color: rgba(0, 0, 0, 0)</string>
     </property>
     <property name="text">
      <string>TODAY:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="energy_logo">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>100</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(&quot;./img/energy_logo.png&quot;);border-radius: 0px; background-color: rgba(0, 0, 0, 0)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_TodayEnergy_unit">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>90</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(&quot;&quot;); border-radius: 0px; background-color: rgba(0, 0, 0, 0)</string>
     </property>
     <property name="text">
      <string>KWH</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_TotalEnergy_unit">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>175</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(&quot;&quot;); border-radius: 0px; background-color: rgba(0, 0, 0, 0)</string>
     </property>
     <property name="text">
      <string>KWH</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QWidget</extends>
   <header location="global">QtCharts/QChartView</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
